etc/docker/Dockerfile.nvcr-triton-containers

```diff
@@ -0,0 +1,78 @@
+ARG TRITON_VERSION=24.06
```

Comment:
This makes it possible to override the triton version on the command line with `--build-arg TRITON_VERSION=xx.yy`.